From 66a1964f40bba4903c254da135cf832d81ebdc4d Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 22:57:43 +0800
Subject: [PATCH 32/58] Revert "drm/amd/amdgpu/amdgpu_cs: fix refcount leak of
 a dma_fence obj"

This reverts commit 927beb05aaa429c883cc0ec6adc48964b187e291.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
index 867fcee6b0d3b..12598a4b5c788 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
@@ -1484,7 +1484,6 @@ int amdgpu_cs_fence_to_handle_ioctl(struct drm_device *dev, void *data,
 		return 0;
 
 	default:
-		dma_fence_put(fence);
 		return -EINVAL;
 	}
 }
-- 
2.27.0

